### PR TITLE
Add compilation of Tokenizer

### PR DESCRIPTION
**Pull request by jsenellart**

### CODE REVIEW

File: frameworks/opennmt_lua/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
+	make install
+
+ARG OPENNMT_LUA_URL
+ENV OPENNMT_LUA_URL=${OPENNMT_LUA_URL:-https://github.com/OpenNMT/OpenNMT.git}
+ARG OPENNMT_LUA_REF
+ENV OPENNMT_LUA_REF=${OPENNMT_LUA_REF:-master}
+ARG OPENNMT_TOKENIZER_REF
+ENV OPENNMT_TOKENIZER_REF=${OPENNMT_TOKENIZER_REF:-master}
+
+RUN git clone --depth 1 --branch ${OPENNMT_LUA_REF} --single-branch ${OPENNMT_LUA_URL} /root/opennmt
```
Comment (guillaumekln) [inline frameworks/opennmt_lua/Dockerfile]: The newest `opennmt/opennmt` image already includes the OpenNMT code. Is that required?

File: frameworks/opennmt_lua/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,60 @@
+FROM ubuntu:14.04 as opennmt_builder
```
Comment (guillaumekln) [inline frameworks/opennmt_lua/Dockerfile]: This should be `ubuntu:16.04` as it is the base image of `opennmt/opennmt:latest`.

File: frameworks/opennmt_lua/Dockerfile [hunk 3]
```diff
@@ -1,14 +1,60 @@
...
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        cmake \
+        wget \
+        gcc \
+        git \
+        libboost-python-dev \
+        python-dev
+
+RUN wget http://www.cmake.org/files/v3.2/cmake-3.2.2.tar.gz && \
```
Comment (guillaumekln) [inline frameworks/opennmt_lua/Dockerfile]: This is not required if using Ubuntu 16.04.

**Comment by jsenellart** [comment]
new version with the last opennmt image!

**Pull request merged**